src/Styleguide/Artwork/Banner.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import styled from "styled-components"
+import { Col, Row } from "../Elements/Grid"
+import { Avatar } from "../Elements/Avatar"
+
+const Flex = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+export class Banner extends React.Component {
+  render() {
+    return (
+      <Row>
+        <Col sm={8}>
+          <Flex>
+            <Avatar size="110px" src="https://picsum.photos/110/110/?random" />
```

Comment:
I know we are building a static page for now but wondering if/where/how we would deal with possibility of image is not present here?

Not sure it's a very likely scenario in this case, but it will be very much likely if we display gallery logo/artist image in this avatar style.